### ISSUE TITLE
Adding a cuboid object to a datapoint requires extra information

### ISSUE THREAD

**Issue by anaoum** [redbrick-ai/redbrick-sdk#184]
When using the `put_tasks` SDK method to update an existing task, we want to add a cuboid to a series. We supply the following information:

```
{
    point1: {i: 34, j: 42, k: 8},
    point2: {i: 38, j: 40, k: 7},
    category: "object_type_1",
    attributes: {},
}
```
Expectation: the task is updated with a new cuboid object.

Actual: the API throws the following error:
```
Error: 'absolutePoint1' is a required property at 0->series->0->cuboids->0            
Docs: https://docs.redbrickai.com/python-sdk/format-reference
```

I understand that _exported_ cuboids should have absolute points, but I am not quite sure why they are required on _import_. In fact, it seems that the provided values in absolute points do not affect how the cuboid renders in the interface once uploaded, so it seems like the server is overwriting these anyway.

**Comment by pritamrungta** [comment]
Thanks @anaoum for submitting the issue. We have triaged it accordingly and will let you know once the fix is released.

**Comment by pritamrungta** [comment]
This has been fixed with sdk@v2.18.7 (with redbrick@v1.1.5)
